.github/workflows/main.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, "3.10"]
```

Comment:
I will try changing the gemmi version, thanks!

Regarding string notation, I first tried the way you say, but that prompted the unittests to use Python 3.1 instead of 3.10! Unless there is another way around it, this is the solution I found.

Reply:
Removing the gemmi version solved it. I am merging now.
Thanks!